### DISCUSSION TITLE
Any way to use ValueListenableBuilder like Hive?

### DISCUSSION THREAD

**Discussion by arbile26** [isar/isar#1335] [Help]
I'm coming from Hive to Isar. I would like to know whether there is any way to listen a specific collection using `ValueListenableBuilder` and do UI updates?

**Comment by giga10** [comment]
https://isar.dev/watchers.html

**Reply by arbile26** [reply]
> https://isar.dev/watchers.html

Thanks for sharing this.
I saw it but it's not mentioned how use it with `valueListenableBuilder` widget.

**Reply by sbenmeddour** [reply]
use a streambuilder
